tests/unit/tornado_tests.py

```diff
@@ -30,3 +35,41 @@ class TornadoConnectionTests(unittest.TestCase):
     def test_tornado_connection_call_parent(self, mock_init):
         obj = tornado_connection.TornadoConnection()
         mock_init.called_once_with(None, None, False)
+
+    def test_client_properties_default(self):
```

Comment:
Similarly to my `blocking_connection_test` feedback, this layer's unit test should avoid duplicating the logic from `connection_test` by simply testing to make sure that the expected `connection_properties` value was passed to the next layer (`BaseConnection` constructor)